deploy/crds/pulpproject_v1alpha1_pulp_cr.travis.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     access_mode: "ReadWriteOnce"
     # We have a little over 40GB free on Travis VMs/instances
     size: "40Gi"
-  database_connection:
+  database_settings:
```

Comment:
As we discussed, as soon as you update this, you'll need to update the plugin-template:
https://github.com/pulp/plugin_template/blob/ff8d62e2b9c7e42c3607977ae79617b70bb2bd41/templates/travis/.travis/install.sh.j2#L108

Reply:
I updated the other files.

Reply:
Also, remind me why we have database_settings renamed from database_connection?

By doing a backwards-incompatible change, we may now need to bump the CR API from `pulpproject.org/v1alpha1`. I'm researching whether this is necessary, but not renaming it would be easier.

Reply:
1. Per our discussion, we can rename it later before we make the operator GA. And we can copy the old dict to the new dict at that time.
2. You renamed it back in the other files, but not this one.

Reply:
updated.